### PR TITLE
fix issue(#49883) Add selector example

### LINKED ISSUE #49883: oc get pods --selector help page should give example of how to use multple selectors
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
> /kind bug
 /kind feature

**What happened**:
The current man/help page for --selector is hard to understand the correct usage.  `man kubectl get` should have an example with comma-delimited use-case, and that multiple --selector (if we can detect it) should have an error message

**What you expected to happen**:
Current help page shows the following.  
`-l, --selector='': Selector (label query) to filter on  `
It would be good to give an example of how to use multiple selectors `oc get pods --selector foo=blah, bar=yeah`  or flag an warning/error if user gave multiple --selector in the command 

**How to reproduce it (as minimally and precisely as possible)**:
`oc get pods  -l component=curator -l provider=openshift`

**Anything else we need to know?**:
none
**Environment**:
- Kubernetes version (use `kubectl version`):
kubernetes v1.3.0+52492b4
- Cloud provider or hardware configuration**:
Openstack
- OS (e.g. from /etc/os-release):
PRETTY_NAME="Red Hat Enterprise Linux Server 7.3 (Maipo)"
- Kernel (e.g. `uname -a`):
Linux openshift-103.lab.sjc.redhat.com 3.10.0-514.10.2.el7.x86_64 #1 SMP Mon Feb 20 02:37:52 EST 2017 x86_64 x86_64 x86_64 GNU/Linux
- Install tools:
- Others:

@kubernetes/sig-cli

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:
fixes #49883

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/annotate.go [hunk 1]
```diff
@@ -130,8 +130,8 @@ func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmdutil.AddPrinterFlags(cmd)
 	cmd.Flags().Bool("overwrite", false, "If true, allow annotations to be overwritten, otherwise reject annotation updates that overwrite existing annotations.")
 	cmd.Flags().Bool("local", false, "If true, annotation will NOT contact api-server but run locally.")
-	cmd.Flags().StringP("selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.")
-	cmd.Flags().Bool("all", false, "Select all resources in the namespace of the specified resource types")
+	cmd.Flags().StringP("selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2 -l key3=value3)")
+	cmd.Flags().Bool("all", false, "select all resources in the namespace of the specified resource types")
```
Comment (thockin) [inline pkg/kubectl/cmd/annotate.go]: Why does this change the capitalization?

Comment (zjj2wry) [inline pkg/kubectl/cmd/annotate.go]: not notice change the capitalization when rebase, updated back. i want add some e.g for users when they use selector. thanks

**Pull request merged**